README.md

```diff
@@ -53,6 +53,7 @@ are designated with a ✅, and hosted projects with a 💙.
 ### React Native
 
 - 💙 [MapLibre React Native](https://github.com/maplibre/maplibre-react-native) - A MapLibre module for React Native (including Expo support).
+- [Expo MapLibre native + web demo](https://github.com/Jaza/expo-maplibre-native-plus-web-demo) - Demo Expo app using [maplibre-react-native](https://github.com/maplibre/maplibre-react-native) for native, and falling back to [react-map-gl](https://github.com/visgl/react-map-gl) with [maplibre-gl-js](https://github.com/maplibre/maplibre-gl-js) for web.
```

Comment:
This is under the binding for react native, is this project a binding project?

Reply:
@HarelM no, it's not a binding, but it's a demo of how to use a react native binding (and fall back to a web binding), so I felt that "Bindings -> React Native" was the most suitable of the existing sections to put it under. Anyway, I've updated this PR, added a new section "Demos / Examples", hope that helps.